istio/istio-install/base/istio-noauth.yaml

```diff
@@ -15839,6 +15880,12 @@ metadata:
     heritage: Tiller
     release: istio
 spec:
+  selector:
+    matchLabels:
+      app: jaeger
+      chart: tracing
+      heritage: Tiller
```

Comment:
nit: Here and everywhere else get rid of heritage label in selector

Reply:
@jlewi Removal of Tiller has been done incrementally in this PR #378